tox.ini

```
@@ -1,6 +1,7 @@
 [tox]
 envlist = py{37,38,39,310},coverage,lint,typecheck,format
 isolated_build = true
+allowlist_externals = *
```

```suggestion
allowlist_externals = poetry
```
should this just be poetry?

---

Tried in https://github.com/argoproj-labs/hera-workflows/pull/389/commits/f6f80aef9a5fe0061291f683c21b8f834a8f4bd8 and it did not seem to work, which is why I wanted to give "*" a go 🤔